src/net.h

```diff
@@ -351,6 +354,7 @@ class CConnman
     unsigned int nSendBufferMaxSize;
     unsigned int nReceiveFloodSize;
 
+    bool m_relay_txes;
```

Comment:
Is it needed? It gets set during `Start()` and isn't used before. Or is it a matter of good form?